enhancements/cluster-logging/cluster-logging-elasticsearch-proxy.md

```diff
@@ -0,0 +1,121 @@
+---
+title: cluster-logging-elasticsearch-proxy
+
+authors:
+  - "@jcantrill"
+
+reviewers:
+  - "@rmeggins"
+  - "@ewolintez"
+  - "@bparees"
+
+approvers:
+  - "@rmeggins"
+  - "@ewolintez"
+
+creation-date: 2019-09-19
+last-updated: 2019-09-19
+status: provisional|implementable|implemented|deferred|rejected|withdrawn|replaced
+see-also: []
+replaces: []
+superseded-by: []
+---
+
+# cluster-logging-elasticsearch-proxy
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift/docs]
+
+## Summary
+
+The `elasticsearch-proxy` is a component that fronts Elasticsearch deployments for OpenShift Logging and is a replacement to the existing plugin `openshift-elasticsearch-plugin`. It provides authentication and authorization to control access to Elasticsearch.  Additionally, it handles logic specific to OpenShift logging to control access to stored data.
```

Comment:
I would add "This is the component that provides multi-tenancy access for Kibana, which allows regular users to see logs only from namespaces to which they belong, and allows admins to see logs from all namespaces and infrastructure/system logs."